Python/ceval_gil.c

```diff
@@ -1192,6 +1192,69 @@ _PyEval_DisableGIL(PyThreadState *tstate)
 }
 #endif
 
+#if defined(Py_REMOTE_DEBUG) && defined(Py_SUPPORTS_REMOTE_DEBUG)
+// Note that this function is inline to avoid creating a PLT entry
+// that would be an easy target for a ROP gadget.
+static inline void run_remote_debugger_script(const char *path)
+{
+    if (0 != PySys_Audit("remote_debugger_script", "s", path)) {
```

Comment:
Is there any reason of swapping the `!=`?

Reply:
Not with this one in particular, but this is https://en.wikipedia.org/wiki/Yoda_conditions